File: Programmers/Lv0/문자열묶기.py
from collections import Counter

def solution(strArr):
    lenArr = []
    for str in strArr:
        lenArr.append(len(str))
    counts = Counter(lenArr)
    most_common_element, count = counts.most_common(1)[0]
    return count

# 매핑방법
def solution(strArr):
    data = [0] * 31
    for x in strArr:
        data[len(x)] += 1
    return max(data)

File: Programmers/Lv0/문자열묶기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<string> strArr) {
    vector<int> data(31, 0);
    for (size_t idx = 0; idx < 31; idx++)
        data[idx] = 0;
    for (string str : strArr)
        data[str.length()] += 1;
    auto max_it = max_element(data.begin(), data.end());
    return *max_it;
}